client/src/modules/cash/cashboxes/cashboxes.html

```diff
@@ -3,7 +3,25 @@
 
     <ol class="headercrumb">
       <li class="static" translate>TREE.ADMIN</li>
-      <li class="title" translate>CASHBOX.TITLE</li>
+
+      <!-- cashbox list -->
+      <li class="title" ng-if="CashCtrl.state.current.name !== 'cashboxes.edit' && CashCtrl.state.current.name !== 'cashboxes.create'" translate>CASHBOX.TITLE</li>
```

Comment:
This is the right idea :+1:

I wonder if it would be clear to make these functions for code clarity?  i.e.
```html
<li class="title" ng-if="CashCtrl.isEditState()">
  <!-- write some code -->
</li>

<li ng-if="CashCtrl.isCreateState">
</li>
```
That way, the HTML is clearly readable?

Maybe this will only move the readability issues to the JS controller... What do you think?